tests/test_orchestrate.py

```diff
@@ -43,469 +87,166 @@ def test_perform_actions_single_parameter_exit(capsys):
-    assert "GatorGrader" in captured.out
-    assert counted_newlines == 7
-    assert exit_code == 1
-
-
-# pylint: disable=redefined-outer-name
-# pylint: disable=bad-continuation
-def test_perform_actions_display_welcome_and_ready_check_regex_command(
-    capsys, reset_results_dictionary
-):
-    """Check the argument verification, messages, and continue for regex output."""
-    chosen_arguments = ["--command", "ls", "--regex", r"G\S{4}G\S{4}r", "--count", "1"]
-    exit_code = orchestrate.check(chosen_arguments)
-    captured = capsys.readouterr()
-    counted_newlines = captured.out.count("\n")
-    assert "GatorGrader" in captured.out
-    assert "regular expression" in captured.out
-    assert "Found 0 match(es)" in captured.out
-    assert counted_newlines == 7
-    assert exit_code == 1
 
-
-# pylint: disable=redefined-outer-name
-# pylint: disable=bad-continuation
-def test_perform_actions_display_welcome_and_ready_check_count_command(
-    capsys, reset_results_dictionary
-):
-    """Check the argument verification, messages, and continue."""
-    chosen_arguments = ["--command", "ls", "--count", "2"]
-    exit_code = orchestrate.check(chosen_arguments)
-    captured = capsys.readouterr()
-    counted_newlines = captured.out.count("\n")
-    assert "GatorGrader" in captured.out
-    assert counted_newlines == 6
-    assert exit_code == 0
-
-
-# pylint: disable=redefined-outer-name
-# pylint: disable=bad-continuation
-def test_perform_actions_display_welcome_and_ready_check_count_command_json(
-    capsys, reset_results_dictionary
-):
-    """Check the argument verification, messages, and continue for JSON output."""
-    chosen_arguments = ["--nowelcome", "--command", "ls", "--count", "2", "--json"]
-    exit_code = orchestrate.check(chosen_arguments)
-    captured = capsys.readouterr()
-    counted_newlines = captured.out.count("\n")
-    assert "GatorGrader" not in captured.out
-    assert counted_newlines == 2
-    assert exit_code == 0
-
-
-# pylint: disable=redefined-outer-name
-# pylint: disable=bad-continuation
-def test_perform_actions_display_welcome_and_ready_check_command(
-    capsys, reset_results_dictionary
-):
-    """Check the argument verification, messages, and continue."""
-    chosen_arguments = ["--command", "ls", "--executes"]
-    exit_code = orchestrate.check(chosen_arguments)
-    captured = capsys.readouterr()
-    counted_newlines = captured.out.count("\n")
-    assert "GatorGrader" in captured.out
-    assert counted_newlines == 6
-    assert exit_code == 0
-
-
-# pylint: disable=redefined-outer-name
-# pylint: disable=bad-continuation
-def test_perform_actions_display_welcome_and_ready_check_command_not_working(
-    capsys, reset_results_dictionary
-):
-    """Check the argument verification, messages, and continue."""
-    chosen_arguments = ["--command", "false", "--executes"]
-    exit_code = orchestrate.check(chosen_arguments)
+@pytest.mark.parametrize(
```

Comment:
This should probably use more than one check, just to ensure that there aren't weird problems.

Reply:
I added several new inputs to this test!